CHANGELOG.md

```diff
@@ -19,12 +25,14 @@ invoke changelog
 * Change auto changelog tool ([#30](https://github.com/eclipse/kiso-testing/issues/30))
 #### Continous integration
 
+* add skip lock for pipenv to prevent hangup
 * github-action for codecov ([#27](https://github.com/eclipse/kiso-testing/issues/27))
 #### Others
 
+* add skip lock for pipenv to prevent hangup
 * github-action for codecov ([#27](https://github.com/eclipse/kiso-testing/issues/27))
```

Comment:
This feels like a duplicate from above ^^

Reply:
yup, 
The file is managed entirely by auto-changelog.

Makes no sense to cleanup this file manualy.
Maybe this stuff will be fixt in future releases.

Reply:
Can we not find out why?

Reply:
Suggestion would be to do it in a different task.
Its the python auto-changelog tool which then needs to be debugged.

The file will always be complete overwritten so we don't have to fix it in this task. 

IMO the release is more important.